src/library_html5_webgl.js

```diff
@@ -14,8 +14,9 @@ var LibraryHtml5WebGL = {
     var len = arr.length;
     var writeLength = dstLength < len ? dstLength : len;
     var heap = heapType ? HEAPF32 : HEAP32;
+    dst = {{{ getHeapOffset('dst', 'float') }}};
```

Comment:
I guess this works because the offset is the same for float and i32 but maybe worth a comment?